backend/src/Controller/UserController.php

```diff
@@ -163,4 +163,17 @@ public function deleteUser(Request $request, $id)
 
 
 
+
+    
+    /**
+     * @Route("/logout", name="logout")
+     */
+    public function logout(): Response
+    {
+        localStorage.removeItem('token');
```

Comment:
doesn't make sense, localStorage is not a php attribute and we're not using symfony to render our frontend so it doesn't make sense to redirect him to a login route